app/src/main/java/com/javahelp/frontend/domain/user/register/IRegisterInput.java

```diff
@@ -0,0 +1,22 @@
+package com.javahelp.frontend.domain.user.register;
```

Comment:
This is already done in `[Backend] Register User Use Case`, or is there anything else specific for this on `Frontend`?

Reply:
I think this is included again so that the frontend and backend can do the work at the same time. Don't know what the specific reasons for this but I did this for my delete account as well.

Reply:
Um Johnny, interesting thought.

Reply:
The backend and frontend should be separate so we cant reuse an interface from the backend in the frontend.